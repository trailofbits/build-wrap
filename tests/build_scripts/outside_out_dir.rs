use std::{env::var_os, fs::write, path::PathBuf};

fn main() {
    let out_dir = var_os("OUT_DIR").unwrap();
    let path = PathBuf::from(out_dir).parent().unwrap().to_path_buf();
    write(path.join("OUTSIDE_OUT_DIR"), "x").unwrap();
}
