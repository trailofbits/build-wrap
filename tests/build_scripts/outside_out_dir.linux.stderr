message: "Read-only file system"
